web/router.ex

```diff
@@ -46,6 +46,9 @@ defmodule Pairmotron.Router do
     pipe_through [:browser, :authenticate] # Use the default browser stack
 
     get "/pairs", PageController, :index
+    get "/profile", ProfileController, :show
+    get "/profile/edit", ProfileController, :edit
+    put "/profile/update/:id", ProfileController, :update
```

Comment:
So I'd love some advice on this, I couldn't get this route to work without specifying `/:id` in the route. I would've liked to just exclude it, as it always updated the currently logged in user. the `id` parameter is completely ignored and is unnecessary.